src/js/components/search/modals/fullDownload/FullDownloadModal.jsx

```diff
@@ -39,18 +39,19 @@ export default class FullDownloadModal extends React.Component {
         this.hideModal = this.hideModal.bind(this);
     }
 
-    componentWillReceiveProps(nextProps) {
+    componentDidUpdate(nextProps) {
```

Comment:
Thanks for updating this method! Could you use `prevProps` for the comparison instead of `nextProps`?

Reply:
fixed and updated branch!